src/components/Hero/index.tsx

```diff
@@ -168,6 +173,30 @@ const HeroButton: React.FunctionComponent<IHeroButton> = ({ title, icon, href, c
   return <div className="sm:w-1/2">{elem}</div>;
 };
 
+const HeroImage: React.FunctionComponent<IHeroImage> = ({ className, section = true, src, alt }) => {
+  if (!alt && !src) {
```

Comment:
```suggestion
  if (!src) {
```

Who cares if there's an `alt`?